kayenta-integration-tests/src/test/resources/prometheus.yml

```diff
@@ -0,0 +1,8 @@
+global:
+  scrape_interval: 1s
+
+scrape_configs:
+  - job_name: kayenta-test-metrics
+    metrics_path: /prometheus
+    static_configs:
+      - targets: ['host.testcontainers.internal:8081']
```

Comment:
Wow, nice :) When I was adding this feature, I was thinking about Selenium & friends, but I'm happy to see it used for other cases too  👍

Reply:
@bsideup I was really glad to find this feature! Thank you! `host.docker.internal` as you know is available only on mac and windows :(

Reply:
Even if they add it to the native Docker, sometimes your "localhost" is not Docker's host (think remote Docker, or when you're running your tests inside a container already)

Testcontainers' approach works everywhere because we start a reverse port forwarding SSH session to the SSH server running as a container inside the target Docker environment :)